pkg/cli/cli.go

```diff
@@ -80,7 +80,7 @@ func runConduitCmdWithConfig(args *data.Args) error {
 	}
 
 	ctx := context.Background()
-	pipeline, err := pipeline.MakePipeline(ctx, pCfg, logger)
```

Comment:
No functional changes here, but removing the shadowing of the package by the variable.